### ISSUE TITLE
Could the authors please explain why, when testing on the COVID-19 dataset, your test set uses the entire dataset?

### ISSUE THREAD

**Issue by xizhengxiang** [nguyenpbui/MMI-UNet#5]
Could the authors please explain why, when testing on the COVID-19 dataset, your test set uses the entire dataset? You did not specifically split a dedicated test set, but instead reused data from the training set and validation set.

**Comment by nguyenpbui** [comment]
For a fair comparison, I followed previous work (LViT) to split the COVID-19 dataset.

**Comment by xizhengxiang** [comment]
I get it, a great innovation!
Thank you for your work!
